code/modules/mob/living/simple_animal/friendly/drone/drones_as_items.dm

```diff
@@ -14,7 +14,7 @@
 	var/drone_type = /mob/living/simple_animal/drone //Type of drone that will be spawned
 	var/seasonal_hats = TRUE //If TRUE, and there are no default hats, different holidays will grant different hats
 	var/static/list/possible_seasonal_hats //This is built automatically in build_seasonal_hats() but can also be edited by admins!
-
+	layer = BELOW_MOB_LAYER
```

Comment:
Move this above the new vars, leaving empty space between the vars and the initialize proc.

Reply:
i don't wanna